ref/appliance-command-reference/monitor.md

```diff
@@ -0,0 +1,50 @@
+The `monitor` command allows managing the monitoring dashboard. You can expose or unexpose the monitoring dashboard using the subcommands. By default, the monitoring dashboard is exposed.
+
+## Usage
+
+    anbox-cloud-appliance monitor <subcommand>
```

Comment:
@morphis , correct me if I was wrong, we've dropped the monitor command for the upcoming minor release(1.20.0), if so, we're going to drop this once 1.20.0 is out?

Reply:
Lets keep it until 1.20 is out and then we can remove it.